spring-cloud-skipper-shell-commands/src/main/java/org/springframework/cloud/skipper/shell/command/support/ShellUtils.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.skipper.shell.command.support;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.springframework.boot.ApplicationArguments;
+
+/**
+ * Various utility functions for a skipper shell.
+ *
+ * @author Janne Valkealahti
+ *
+ */
+public abstract class ShellUtils {
+
+	private final static List<String> helpArgs = Arrays.asList("-h", "-help", "--help", "help");
```

Comment:
I think we could add `--h` as well. I can fix it when merging.

Reply:
To support `--h` we need `h` as well as the option. I think it is OK to have it at least as a convenient :-)